### PR TITLE
Ivory 249/mercury package

### PR DESCRIPTION
**Pull request by mihai-ro**

### CODE REVIEW

File: packages/mercury/src/lib/HeroSearch/HeroSearch.molecule.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  onSearch: requestSearch,
+  subtitle,
+  title,
+  theme,
+  ...props
+}) => {
+  const [searchTerm, setSearchTerm] = useState('')
+  const _handleValueChange = ev => setSearchTerm(ev.target.value)
+  const _search = ev => {
+    if (!searchTerm.length) {
```
Comment (BvmvB) [inline packages/mercury/src/lib/HeroSearch/HeroSearch.molecule.js]: Shouldn't we `preventDefault()` in all cases?

**Comment by mihai-ro** [comment]
@all-contributors please add @mihai-ro for infrastructure, code
@all-contributors please add @msal96  for code
@all-contributors please add @dan-puscaru for design

**Comment by mihai-ro** [comment]
@all-contributors please add @mihai-ro  for code, documentation, architecture and infra
@all-contributors please add @mihai-ro for infrastructure
@all-contributors please add @msal96  for code
@all-contributors please add @dan-puscaru for design

**Comment by mihai-ro** [comment]
@all-contributors please add @dan-puscaru for design

**Comment by mihai-ro** [comment]
@all-contributors please add @BvmvB for fundingFinding, ideas, infra, projectManagement, review

**Comment by mihai-ro** [comment]
@all-contributors please add @msal96  for code

**Comment by mihai-ro** [comment]
@all-contributors please add @mihai-ro for code, content, doc, examples, ideas, infra, maintenance, plugin, projectManagement, review, tool, tutorial, talk

**Comment by mihai-ro** [comment]
@all-contributors please add @msal96  for maintenance

**Pull request merged**